solr/core/build.gradle

```diff
@@ -139,12 +139,12 @@ dependencies {
   implementation 'org.apache.logging.log4j:log4j-core'
   implementation 'org.apache.logging.log4j:log4j-slf4j-impl'
 
-  implementation 'io.dropwizard.metrics:metrics-core'
-  implementation ('io.dropwizard.metrics:metrics-graphite', {
+  compileOnlyApi 'io.dropwizard.metrics:metrics-core'
```

Comment:
Probably worth a comment about these saying why and that it is referenced in `solr/server/build.grade`.

Reply:
Please feel free to add a comment on the PR/branch; I'm not yet familiar enough with this gradle stuff to put a helpful and accurate comment. But diving into a mystery like this here is a fun way to learn more about gradle.

Reply:
While this change works I assume, I think it's fundamentally wrong to mis-advertise the proper scope just because of our packaging quirks.

Reply:
Nevermind; I get it, it's fine.

Reply:
I pushed a comment in 85e3c74 and slightly rearranged core/build.gradle to match the structure in server/build.gradle. Hopefully that makes it easier to not screw it up next time :D